test/input_gen/genModelsRecurrent_v2.py

```diff
@@ -197,147 +199,165 @@ def forward(self, inputs, labels):
         name="lstm_stacked",
     )
 
+    unroll_for, num_lstm, state_num, batch_size, unit, feature_size, iteration, hidden_state_zoneout_rate, cell_state_zoneout_rate = [2, 1, 2, 1, 2, 2, 2, 0.0, 0.0]
     record_v2(
-        ZoneoutLSTMStacked(batch_size=1, unroll_for=2, num_lstm=1, hidden_state_zoneout_rate=0.0, cell_state_zoneout_rate=0.0),
-        iteration=2,
-        input_dims=[(1, 2)],
-        label_dims=[(1, 2, 2)],
+        ZoneoutLSTMStacked(batch_size=batch_size, unroll_for=unroll_for, num_lstm=num_lstm, hidden_state_zoneout_rate=hidden_state_zoneout_rate, cell_state_zoneout_rate=cell_state_zoneout_rate),
```

Comment:
recommend to use functools partial if this setup is being repeatedly used. For now it looks fine :)